.github/workflows/publish_draft_releases.yml

```diff
@@ -571,6 +572,249 @@ jobs:
+          path: /home/runner/.local/share/calamari-pc/${{ matrix.chain-spec.id }}-${{ steps.create-chainspec.outputs.short-sha }}-launch-config.json
+      - uses: actions/checkout@v2
+        with:
+          repository: paritytech/polkadot-launch
+          path: polkadot-launch
+      - uses: actions/checkout@v2
+        with:
+          repository: Manta-Network/Dev-Tools
+          path: dev-tools-rococo
+      - uses: actions/checkout@v2
+        with:
+          repository: Manta-Network/Dev-Tools
+          path: dev-tools-calamari
+      - name: launch testnet
+        run: |
+          cd ${{ github.workspace }}/polkadot-launch
+          yarn install
+          yarn build
+          pm2 start dist/cli.js \
+            --name polkadot-launch \
+            --output ${{ github.workspace }}/polkadot-launch-for-${{ matrix.chain-spec.id }}-stdout.log \
+            --error ${{ github.workspace }}/polkadot-launch-for-${{ matrix.chain-spec.id }}-stderr.log \
+            --no-autorestart \
+            -- $HOME/.local/share/calamari-pc/${{ matrix.chain-spec.id }}-${GITHUB_SHA:0:7}-launch-config.json
+      - name: init measure-block-time rococo
+        run: |
+          cd ${{ github.workspace }}/dev-tools-rococo/measure-block-time
+          yarn install
+          pm2 start index.js \
+            --name measure-block-time-rococo-relay-for-${{ matrix.chain-spec.id }} \
+            --output ${{ github.workspace }}/measure-block-time-rococo-relay-for-${{ matrix.chain-spec.id }}-stdout.log \
+            --error ${{ github.workspace }}/measure-block-time-rococo-relay-for-${{ matrix.chain-spec.id }}-stderr.log \
+            --no-autorestart \
+            -- \
+            --address=ws://127.0.0.1:9921
+      - name: init measure-block-time calamari
+        run: |
+          cd ${{ github.workspace }}/dev-tools-calamari/measure-block-time
+          yarn install
+          pm2 start index.js \
+            --name measure-block-time-${{ matrix.chain-spec.id }} \
+            --output ${{ github.workspace }}/measure-block-time-${{ matrix.chain-spec.id }}-stdout.log \
+            --error ${{ github.workspace }}/measure-block-time-${{ matrix.chain-spec.id }}-stderr.log \
+            --no-autorestart \
+            -- \
+            --address=ws://127.0.0.1:9921
+      - uses: actions/checkout@v2
+        with:
+          repository: Manta-Network/Manta
+          path: Manta
+      - name: run test suites
+        run: |
+          # TODO: implement moonbeam-like js test suite triggers here
+          sleep 120
+          cd ${{ github.workspace }}/Manta/tests
+          yarn install
+          yarn
+          yarn correctness_test --address=ws://127.0.0.1:9921 --exit
+          echo $?
+          if [ $? != 0 ]; then echo "Ledger RPC correctness test failed!"; exit 1; fi
+          yarn performance_test --address=ws://127.0.0.1:9921 --exit
```

Comment:
I don't see a lot of reasons to put the performance test in CI. Since the result of the performance test is a score rather than pass/not pass.

Reply:
I changed it to to check that it is below an expected sync time https://github.com/Manta-Network/Manta/pull/681/files#diff-19d51960f59786f9c40532e38e3a902c64be0e21e3b84a46702bcb0718a20605R20

Reply:
Useful as a sanity check mainly so we're sure it hasn't regressed for whatever reason.